### PR TITLE
finished challenge

### PR DESCRIPTION
**Pull request by isabelthedesigner**
finished!

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+footer {
+	font-family: 'Cutive Mono', monospace;
+	text-align: center;
+	padding-top: 50px;
+	padding-bottom: 20px;
+
+}
+
```
Comment (zoeames) [inline css/styles.css]: what are all these empty lines for?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (zoeames) [inline index.html:2]: check your indentation in this file.  head and body should be tabbed under html

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+			<br>
+			<section>
+			<h3>PROJECT #2:</h3>
+			<p>Enamel pin iceland tofu, mumblecore sartorial vape kitsch normcore bicycle rights locavore street art messenger bag shabby chic poke single-origin coffee. Vexillologist plaid godard food truck, butcher leggings viral. Waistcoat meggings actually thundercats tousled, fap 3 wolf moon.</p>
+			</section>
+
+	</article>
+
+	<footer>
+		Copyright &copy; 2017 Baddest Blues Design | <a href="mailto:isabel@baddestbluesdesign.com">isabel@baddestbluesdesign.com</a> | <a href="https://github.com/isabelramos">GitHub</a>
```
Comment (zoeames) [inline index.html]: how about using icons here instead of links?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+		<p class="links-para"><a href="https://hipsum.co" target="_blank">Hipster Ipsum</a>
+		<br><a href="https://developer.mozilla.org/en-US/" target="_blank">Mozilla Developer Network</a>
+		<br><a href="https://creativemarket.com/" target="_blank">Creative Market</a></p>
+	</article>
+
+	<article>
+		<h2>PROJECTS IN THE WORKS:</h2>
+		
+			<section>
+			<h3>PROJECT #1:</h3>
```
Comment (zoeames) [inline index.html]: are you going to have screen shots of projects next to the text? (the correct answer here is yes)  if so stick a fake project screenshot into the place you would put it.

**Comment by zoeames** [comment]
very nice start!  but only 1 commit?  that can be dangerous.  A few comments in the html for cleanup.  Also its a good adea to add comments to your css and html files to break things up.  In the css file this will help break the styles up in a more clear way.  That way if you ever want to change the header styles you know you just look at the stuff under the header section.

**Comment by isabelthedesigner** [comment]
@zoeames

**Comment by zoeames** [comment]
comments look good but what about the other 2 things above?  icons and project screen shots

**Comment by isabelthedesigner** [comment]
i'm not totally sure how to add a comment to all the code that shows the icons / project photos so i screenshot it. let me know if this works!

screen shot 2017-02-12 at 4 41 09 pm
screen shot 2017-02-12 at 4 38 58 pm

**Comment by zoeames** [comment]
@isabelramos Yes that is what i am looking for but I don't see that in your code.  Can you push those changes up?

**Comment by isabelthedesigner** [comment]
weird, i pushed everything up yesterday but i think it only pushed the updated css? not sure but it should work now, i believe!

**Comment by zoeames** [comment]
that looks good! 👍 I will merge for you

**Pull request merged**